azerothcore/mod-account-mounts
# Add Configuration Option for limitrace

## Changes Proposed:
- Added configuration-driven `limitrace` option to the `AccountMounts` module.
- Refactored the code to use the `limitrace` member variable consistently within the `AccountMounts` class.

## Issues Addressed:

-

## SOURCE:

- Internal enhancement request to make the `limitrace` option configurable for better flexibility and easier management.

## Tests Performed:

- Built the project without errors.
- Tested in-game to ensure the configuration options are read correctly and the functionality works as expected.
- Verified on Ubuntu 22.04

## How to Test the Changes:


1. Update the server configuration file to include the new option.
2. Compile and run the server
3. Log in with a player and verify the following:
- If Account.Mounts.LimitRace is set to 1, only mounts from characters on the same team and class should be considered.
- if Account.Mounts.LimitRace is set to 0, it should disregard any class/race limitations.
